ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2331,11 +2373,15 @@ router_bgp:
     neighbor_default:
       encapsulation: < vxlan | mpls >
       next_hop_self_source_interface: < source interface >
+      next_hop_self_received_evpn_routes:
+        enable: < true | false >
+        inter_domain: < true | false >
     peer_groups:
       < peer_group_name >:
         activate: < true | false >
         route_map_in: < route_map_name >
         route_map_out: < route_map_name >
+        domain_remote: < true | false >
```

Comment:
please implement for neighbors as well.

Reply:
EGW2(config-router-bgp)#neighbor 1.1.1.1 next-hop-self ?
  <cr> 

It's only applicable to neighbor default at this stage @ClausHolbechArista

Reply:
There doesn't seem to be a data model for ungrouped neighbors under the evpn AF:

```yaml
address_family_evpn:
    domain_identifier: < string >
    neighbor_default:
      encapsulation: < vxlan | mpls >
      next_hop_self_source_interface: < source interface >
      next_hop_self_received_evpn_routes:
        enable: < true | false >
        inter_domain: < true | false >
    peer_groups:
      < peer_group_name >:
        activate: < true | false >
        route_map_in: < route_map_name >
        route_map_out: < route_map_name >
        domain_remote: < true | false >
    evpn_hostflap_detection:
      enabled: < true | false >
      threshold: < integer >
      window: < integer >
    route:
      import_match_failure_action: < 'discard' >
```

Reply:
Not sure where my comments are going... So we only support the neighbour default * for this knob.

Reply:
The knob claus is referring to is supported in EOS on non-peer-grouped neighbors as well, but we don't have support for non-peer-grouped neighbors under evpn AF in eos_cli_config_gen.